Auth0/SafariWebAuth.swift

```diff
@@ -76,18 +78,31 @@ class SafariWebAuth: WebAuth {
         return self
     }
 
-    func usingImplicitGrant() -> Self {
-        self.usePKCE = false
+    func response(_ response: [ResponseOptions]) -> Self {
+        if !response.contains(.code) { self.usePKCE = false }
```

Comment:
We can probably retire self.userPKCE and replace with a check to see if response contains .code